### PR TITLE
feat: add from=chatroom query and menu item 'View'

### PR DESCRIPTION
**Pull request by jillyratria**
> This project is a forked version of the [Sendbird UI Kit](https://github.com/sendbird/sendbird-uikit-react) to match with the Ruangguru's internal requirements. Therefore, it is not yet set up to accept pull requests from external contributors. But you can always freely create a forked version from this repository to match your own requirements.

## Description Of Changes

- [ X]  show menu item view on file message

Screen Shot 2021-11-01 at 20 23 42

- [ X]  show menu item view on thumbnail message
Screen Shot 2021-11-01 at 20 23 53

- [ X]  add query `from=chatroom` assignment and material open

### CODE REVIEW

File: src/rogu/ui/MaterialMessageItemBody/index.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function MaterialMessageItemBody({
 
   const openMaterial = (): void => {
     if (materialData?.ctaWeb && materialData?.ctaWeb.length > 0){
-      window.open(materialData?.ctaWeb);
+      window.open(`${materialData?.ctaWeb}?from=chatroom`);
```
Comment (suliskh) [inline src/rogu/ui/MaterialMessageItemBody/index.tsx:27]: Mbak @jillyratria, nuwun sewu ini apakah tidak disamakan dengan payload tracker-nya saja mbak? I.e. `obrolan_kelas`

File: src/rogu/ui/MessageInput/index.jsx [hunk 2]
```diff
@@ -124,6 +125,7 @@ const MessageInput = React.forwardRef((props, ref) => {
       } else {
         onSendMessage(trimmedInputValue);
         setInputValue('');
+        elem.style.height = `${LINE_HEIGHT}px`;
```
Comment (suliskh) [inline src/rogu/ui/MessageInput/index.jsx]: I am wondering if `elem.style` will yielding an error because `elem` object is possibly nullish, Mbak?

**Review by suliskh — CHANGES_REQUESTED** [review]

File: src/rogu/ui/MaterialMessageItemBody/index.tsx (hunk 1 above)
Comment (jillyratria) [reply src/rogu/ui/MaterialMessageItemBody/index.tsx:27]: yang di tracker-nya Ibe, pake `chatroom` Kuh. Pake yang `chatroom` dulu. Yang di tracker nya chatroom nanti di-update.

Comment (suliskh) [reply src/rogu/ui/MaterialMessageItemBody/index.tsx:27]: Awraitt mbak jill. Suwunn. Nice work 🔥

File: src/rogu/ui/MessageInput/index.jsx (hunk 2 above)
Comment (jillyratria) [reply src/rogu/ui/MessageInput/index.jsx]: updated ya @suliskh, maacih udah dilihat~~

**Review by suliskh — APPROVED** [review]
Great work!

**Pull request merged**